### PR TITLE
chore: cleanup helm deployments

### PR DESCRIPTION
**Pull request by junminahn**
- clean up helm deployments in a favour of shared helm modules.
- add a keycloak script to get session stats.

### CODE REVIEW

File: helm/keycloak/deploy-eb75ad-dev.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@
 helm repo add sso-charts https://bcgov.github.io/sso-helm-charts
 helm repo update
 
-helm upgrade --install sso-keycloak sso-charts/sso-keycloak -n eb75ad-dev -f values-eb75ad-dev.yaml --version v1.2.3
+helm upgrade --install sso-keycloak sso-charts/sso-keycloak -n eb75ad-dev -f values-eb75ad-dev.yaml --version v1.4.1
```
Comment (thegentlemanphysicist) [inline helm/keycloak/deploy-eb75ad-dev.sh:6]: Will we need to manually change this version on new deployments of keycloak? Or does this reflect the version of keycloak we are installing?

File: helm/keycloak/values-b861c7-test-1.yaml [hunk 2]
```diff
@@ -29,3 +22,27 @@ resources:
...
     cpu: 0.5
     memory: 512M
+
+patroni:
+  install: true
+  replicaCount: 3
+
+  additionalCredentials:
+    - username: ssokeycloak1
+      password:
```
Comment (thegentlemanphysicist) [inline helm/keycloak/values-b861c7-test-1.yaml:32]: How does these passwords get set? do we fill this out on a local branch and then install the helm charts?
If so should we have some kind of place holder? Or does should we reference a custom secret with?

File: helm/keycloak/deploy-eb75ad-dev.sh (hunk 1 above)
Comment (junmin-bcgov) [reply helm/keycloak/deploy-eb75ad-dev.sh:6]: this is the version control; when we deploy newer version of helm chart, we need to explictly update the version here if want to make a change for this release.

File: helm/keycloak/values-b861c7-test-1.yaml (hunk 2 above)
Comment (junmin-bcgov) [reply helm/keycloak/values-b861c7-test-1.yaml:32]: I made the helm template more dynamic so it won't require to create the password secret anymore;
https://github.com/bcgov/sso-helm-charts/blob/b8974afebc7f5cbed2de12cf3431f5e22172f3cf/charts/patroni/values.yaml#L27

Comment (thegentlemanphysicist) [reply helm/keycloak/values-b861c7-test-1.yaml:32]: Ah okay that makes sense, thank you

**Review by thegentlemanphysicist — APPROVED** [review]

**Pull request merged**